### ISSUE TITLE
Sensible defaults for log driver options

### ISSUE THREAD

**Issue by kerihenare** [docker/for-mac#5999]
I started filling out the standard issue template, but I'm not sure it quite fits (and my answers kept sounding a bit passive aggressive).

I've had a reoccurring issue with **Docker for Mac** where the disk image grows quickly to fill the allocated space. I've implemented all recommended best practice methods to avoid such issues including regularly running a script which does a thorough prune of everything it can, and running the `docker/desktop-reclaim-space` image. But I still kept occasionally running into the issue, and the only solution was to run **Clean / Purge data**.

After using @justincormack's **nsenter1** image, I noticed that one of my containers had a 96GB+ log file. I was surprised to find that the default logger `max-size` was **-1**. I don't believe it's a sensible default for **Docker for Mac**, especially given that the log file size is hidden away in the VM and isn't shown anywhere in the GUI.

**TLDR:** Can a sensible default value please be set for `log-opts.max-size`, to avoid excessive disk image size.
